src/Twig/Extension/SonataAdminExtension.php

```diff
@@ -102,6 +100,8 @@ class SonataAdminExtension extends AbstractExtension
 
     /**
      * NEXT_MAJOR: Make $propertyAccessor mandatory.
+     *
+     * @internal
      */
```

Comment:
NEXT_MAJOR:
- Remove the `@internal`
- Remove the $translator
- Remove the $securityChecker
- Make argument mandatory